src/main/scala/dataduggee/model.scala

```diff
@@ -83,3 +83,11 @@ object model {
     }
   }
 }
+
+object error {
+  abstract class DataDuggeeError extends Exception
+
+  case class RequestError(code: Int, message: String) extends Exception {
```

Comment:
Did you mean to extend `DataDuggeeError` here or forgot to delete the `DataDuggeeError`?